app/views/shared/_upcoming_events.html.erb

```diff
@@ -0,0 +1,44 @@
+<section>
+    <h1> Upcoming Events </h1>
+    <h3> See What's Next! </h3>
+    <div class="container marketing">
+      <div class="row">
+          <div class="col-md-4">
+            <div class="panel panel-default">
+              <div class="panel-image"><img src="../../assets/images/stupid_events.jpg" alt=""></div>
+              <div class="panel-body">
+                <h4></h4>
+                <p></p>
+              </div>
+                <div class="text-center">
+                  <a href="" class="btn btn-primary"> Register Now </a>
+                </div>
+            </div>
+          </div>
+          <div class="col-md-4">
+            <div class="panel panel-default">
+              <div class="panel-image"><img src="../../assets/images/stupid_events.jpg" alt=""></div>
+              <div class="panel-body">
+                <h4></h4>
+                <p></p>
+              </div>
+                <div class="text-center">
+                  <a href="" class="btn btn-primary"> Register Now </a>
+                </div>
+            </div>
+          </div>
+          <div class="col-md-4">
+            <div class="panel panel-default">
+              <div class="panel-image"><img src="../../assets/images/stupid_events.jpg" alt=""> </div>
+              <div class="panel-body">
+                <h4></h4>
+                <p></p>
+              </div>
+                <div class="text-center">
+                  <a href="" class="btn btn-primary"> Register Now </a>
+                </div>
+            </div>
+          </div>
+    </div>
```

Comment:
The indentation doesn't look quite right here - could you double check? (Do you have beautify installed in Atom? You will probably need to update the defaults to have two spaces instead of four, but it will help you tidy up your indentation.)